apps/ledger-live-desktop/tests/page/account.page.ts

```diff
@@ -95,5 +98,19 @@ export class AccountPage extends AppPage {
     await this.scrollToOperations();
     await expect(this.lastOperation).toBeVisible();
     await expect(this.operationList).not.toBeEmpty();
+    if (await this.showMoreButton.isVisible()) await this.showMoreButton.click();
+  }
+
+  @step("Expect `derivate address index to be 0` to be visible")
+  async expectAddressIndexToBe0() {
+    await this.settingsButton.click();
+    await this.AdvancedButton.click();
+    const advancedLogsText = await this.accountAdvancedLogs.innerText();
+    if (advancedLogsText !== null) {
+      const advancedLogsJson = JSON.parse(advancedLogsText);
+      expect(advancedLogsJson).toHaveProperty("index", 0);
+    } else {
```

Comment:
could be simplified to something like `avancedLogsText ? expect(advancedLogsJson).toHaveProperty("index", index); : throw`

Reply:
Do we really need to throw an error when null ?
Why not simply asset ?
`const advancedLogsJson = JSON.parse(await this.accountAdvancedLogs.innerText());
expect(advancedLogsJson).toHaveProperty("index", 0);`

Reply:
➕